From b77e341fe7c80cc27a0eb3688b9c5036d952419d Mon Sep 17 00:00:00 2001
From: Lorenzo Pirritano <lpirritano@anaconda.com>
Date: Mon, 1 Jul 2024 09:56:54 +0200
Subject: [PATCH 2/2] fix use spdlog target

---
 src/cc/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/cc/CMakeLists.txt b/src/cc/CMakeLists.txt
index cf16d04..9975cf8 100644
--- a/src/cc/CMakeLists.txt
+++ b/src/cc/CMakeLists.txt
@@ -11,8 +11,7 @@ target_sources(
 target_include_directories(
   BiDirectionalCpp
   PUBLIC $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}>
-         $<BUILD_INTERFACE:${PROJECT_BINARY_DIR}>
-         $<BUILD_INTERFACE:${spdlog_SOURCE_DIR}/include>)
+         $<BUILD_INTERFACE:${PROJECT_BINARY_DIR}>)
 
 target_compile_features(BiDirectionalCpp PUBLIC cxx_std_20)
 
@@ -26,6 +25,7 @@ set_target_properties(
              VERSION ${PROJECT_VERSION})
 
 target_link_libraries(BiDirectionalCpp PRIVATE LEMON::LEMON)
+target_link_libraries(BiDirectionalCpp PRIVATE spdlog::spdlog)
 
 add_library(${PROJECT_NAME}::BiDirectionalCpp ALIAS BiDirectionalCpp)
 
-- 
2.39.1

