From 1d9143fbf975ad9f40233b6c777789e436c2629d Mon Sep 17 00:00:00 2001
From: Lorenzo Pirritano <lpirritano@anaconda.com>
Date: Tue, 16 Jul 2024 14:57:24 +0200
Subject: [PATCH 4/7] do not download swig

---
 CMakeLists.txt | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 16b91f2..9bde30f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -138,14 +138,6 @@ FetchContent_MakeAvailable(spdlog)
 enable_testing()
 include(cpp)
 
-if(WIN32
-   AND (BUILD_PYTHON
-        OR BUILD_JAVA
-        OR BUILD_DOTNET))
-  message(STATUS "Getting SWIG: ...")
-  include(swig)
-  message(STATUS "Getting SWIG: ...DONE")
-endif()
 
 include(python)
 include(dotnet)
-- 
2.39.1

